execution/test_utils.go

```diff
@@ -409,6 +409,35 @@ func AreStreamsEqualNoOrdering(ctx context.Context, stateStorage storage.Storage
 	return nil
 }
 
+func AreStreamsEqualWithOrdering(ctx context.Context, stateStorage storage.Storage, first, second RecordStream) error {
+
+	log.Println("first stream")
+	firstRecords, err := ReadAll(ctx, stateStorage, first)
+	if err != nil {
+		return errors.Wrap(err, "couldn't read first streams records")
+	}
+	log.Println("read first stream")
+
+	log.Println("second stream")
+	secondRecords, err := ReadAll(ctx, stateStorage, second)
+	if err != nil {
+		return errors.Wrap(err, "couldn't read second streams records")
+	}
+	log.Println("read second stream")
+
+	if len(firstRecords) == len(secondRecords) {
```

Comment:
why `==` and not `!=` ?

Reply:
if they are of equal length then error should be `streams don't have different ammount of records` or `streams have equal amount of records`

Reply:
amount* not ammount